erdos/graph.py

```diff
@@ -263,7 +263,8 @@ def _build_refined_op_graph(self):
         for op_id, op_handle in self.op_handles.items():
             op_handle.op_cls.setup_streams(
                 DataStreams(op_handle.input_streams), **op_handle.setup_args)
-
+        for op_id, op_handle in self.op_handles.items():
+            print(op_handle.name, [out.name for out in op_handle.output_streams])
```

Comment:
Can you please improve logging here? Please give the context of what you're logging (e.g., Operator foo has output streams: bar, bar2)

Reply:
Oops that was for debugging forgot to delete it